web/styles/zulip.css

```diff
@@ -1049,6 +1049,110 @@ p.n-margin {
     position: relative;
 }
 
+/* Styles for the overlay screen that appears while uploading files using drag-and-drop */
+.upload-overlay-container {
+    visibility: hidden;
+    position: fixed;
+    display: flex;
+    top: 0;
+    left: 0;
+    width: 100vw;
+    height: 100vh;
+    flex-direction: column;
+    align-items: center;
+    justify-content: center;
+    gap: 16px;
+    /* Semi-transparent background with a blur effect */
+    background: hsl(238deg 35% 15% / 65%);
+    backdrop-filter: blur(8px);
+    transition: all 400ms;
+    opacity: 0;
+    /* Initial low z-index to avoid interfering with other components */
+    z-index: -1;
+}
+
+/* Dynamic class to make the overlay visible during drag events */
+.upload-overlay-visibility {
+    visibility: visible;
+    /* Higher z-index to overlay other components during file upload */
+    z-index: 10000;
+    opacity: 1;
+    pointer-events: all;
+}
+
+.upload-overlay-svg-container {
+    width: 100%;
+    height: 100%;
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    margin: auto;
+}
+
+/* Styles for the animated border SVG */
+.upload-overlay-border {
+    width: calc(100% - 80px);
```

Comment:
Can you leave a comment explaining where `80px` is coming from?

Reply:
I have used these values based on [issue description](https://github.com/zulip/zulip/issues/26565) or [Figma design](https://www.figma.com/file/jbNOiBWvbtLuHaiTj4CW0G/Zulip-Web-App?type=design&node-id=6229-148055&mode=design) given by @terpimost.
Although, I don't the exact reason but I think terpimost can help me with this , so that I can write comment.